charts/budibase/templates/app-service-deployment.yaml

```diff
@@ -212,10 +212,6 @@ spec:
         - name: APP_FEATURES
           value: "api"
         {{- end }}
-        {{- if .Values.globals.sqs.enabled }}
```

Comment:
Don't we need to add the new feature flag env in here?

Reply:
The only users of this so far is us, and we're going to move to using PostHog to flag this on and off, so I decided to just get rid of it. When it comes time for this to be turned on for users, it'll be done by default.